bundle/manifests/argocd-operator.clusterserviceversion.yaml

```diff
@@ -1901,7 +1901,7 @@ spec:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: ENABLE_CONVERSION_WEBHOOK
                   value: "true"
-                image: quay.io/argoprojlabs/argocd-operator@sha256:405e7bde25affb7644c576a7f2d41ba30b88e2ef4f2f11945b920a33517895ce
+                image: quay.io/argoprojlabs/argocd-operator:v0.12.0
```

Comment:
running `make bundle` seems to be automatically replacing the sha with tag. Do we see any issues with this update?

Reply:
I think it should be fine. The CSV is updated with sha during operator release so shouldn't be a issue.